proposals/devcontainer-features-distribution.md

```diff
@@ -131,6 +131,8 @@ A feature can be referenced directly in a user's [`devcontainer.json`](/docs/spe
 
 To aid in feature authorship, or in instances where a feature should not be published externally, individual features can be referenced locally from the project's file tree.
 
-A feature can be referenced directly in a user's [`devcontainer.json`](/docs/specs/devcontainer-reference.md#devcontainerjson) by relative path _inside_ the project directory.  A local feature may not be referenced outside of the project directory (`../` is not allowed), nor is an absolute path allowed.  
+A local feature is referenced in a user's [`devcontainer.json`](/docs/specs/devcontainer-reference.md#devcontainerjson) by relative path. A local feature is always referenced relative to the project root.
+
+A local feature may not be referenced outside of the project's root directory (`../` is not allowed), nor is an absolute path allowed.
```

Comment:
With my above comment `../` would be allowed as long as it does not go outside the project's root folder.

Reply:
Can we define the project root as well? Is that based on the repository root, which implies that all devcontainers need to be inside a git repo, or is there some other definition of project root?

I'd rather be more restrictive to start and then open up the set of valid paths later on if needed based on feedback. It's much easier to broaden the set of allowed paths over time than to narrow the set due to backwards compatibility requirements.

Reply:
The workspace folder (i.e., project folder) is passed to the CLI with `--workspace-folder` and that is used to look up the devcontainer.json (which can be overridden by another CLI option).

I agree on being restrictive. Would we be fine with `.devcontainer/devcontainer.json` not being able to refer to a local feature outside of the `.devcontainer` folder?

Reply:
I think restricting to the `.devcontainer` folder is good to start.  @samruddhikhandale - the `devcontainers/images` repo would need to adapt to this, right?

Reply:
> @samruddhikhandale - the devcontainers/images repo would need to adapt to this, right?

Yes!